src/options.d.ts

```diff
@@ -37,7 +37,5 @@ export {
   NuxtI18nInternalOptions,
   DetectBrowserLanguageOptions,
   RootRedirectOptions,
-  LanguageSwitchedHandler,
-  BeforeLanguageSwitchHandler,
   LanguageSwitchedHandler
```

Comment:
we need to remove this line (`LanguageSwitchedHandler`)
I don't remember why I exported the same type. And typescript doesn't warn us. 😅